CSharp/HelloWorld/HelloWorld.csproj

```diff
@@ -36,8 +36,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>packages\Hyak.Common.1.0.2\lib\net45\Hyak.Common.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Batch">
-      <HintPath>packages\Azure.Batch.2.0.1\lib\net45\Microsoft.Azure.Batch.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Batch, Version=2.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
```

Comment:
not sure. VS added that when I went through the upgrade process for just this package.